changelog.d/4-docs/WPB-1103

```diff
@@ -0,0 +1 @@
+Adding relative URLs to Swagger and OpenApi documentation to reflect the API versions.
```

Comment:
so, previous behavior was that openapi only exposed v0?  hehe, then maybe this should be called a bugfix here?

```suggestion
Fix: support api versions other than v0 in swagger docs.
```

Reply:
They all only exposed V0, but yes I guess that is a valid look at it.